commercetools-models/src/main/java/io/sphere/sdk/taxcategories/ExternalTaxRateDraft.java

```diff
@@ -22,6 +22,8 @@
     @Nullable
     String getState();
 
+    Boolean isIncludedInPrice();
```

Comment:
Should be nullable or default should be set.

Reply:
See comment above and it shouldn't be `@Nullable`, because we only use that annotation for optional fields.